homeassistant/components/insteon/strings.json

```diff
@@ -67,6 +68,12 @@
           "password": "[%key:common::config_flow::data::password%]"
         }
       },
+      "change_plm_config": {
+        "description": "Change the Insteon PLM connection information. You must restart Home Assistant after making this change. This does not change the configuration of the PLM itself.",
```

Comment:
Wouldn't it be enough to reload the config entry instead of restarting Home Assistant? Why don't we do that automatically after updating the config entry?

Reply:
I see that there's no `async_unload_entry` function in the init module. That should probably be added too with a call to close the onsteon connection.